js/form.js

```diff
@@ -0,0 +1,27 @@
+const adForm = document.querySelector('.ad-form');
+const adFieldsets = adForm.querySelectorAll('fieldset');
+const mapFormFilter = document.querySelector('.map__filters');
+const mapSelects = mapFormFilter.querySelectorAll('select');
+const mapFieldsets = mapFormFilter.querySelectorAll('fieldset');
```

Comment:
Ты можешь сразу найти все нужные элементы
```suggestion
const filterForm = mapFormFilter.querySelectorAll('select, fieldset');
```